client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -45,34 +46,36 @@ export default function DesktopNavbar() {
   return (
     <div className="desktop-navbar">
       <NavbarSection inlineCollapsed={collapsed} className="desktop-navbar-logo">
-        <a href="./">
-          <img src={logoUrl} alt="Redash" />
-        </a>
+        <div>
```

Comment:
Because And menu component sees that its children are not a simple html element and tries to pass some custom props to it, and we're getting this:

![image](https://user-images.githubusercontent.com/12139186/91281309-93d8fe80-e790-11ea-96df-ac6432a4df1d.png)

(and many of such errors below)